src/models/generated/Transformer3W.jl

```diff
@@ -0,0 +1,380 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+
+#! format: off
+
+"""
+    mutable struct Transformer3W <: ACBranch
+        name::String
+        available::Bool
+        primary_secondary_arc::Arc
+        secondary_tertiary_arc::Arc
+        primary_tertiary_arc::Arc
+        star_bus::ACBus
+        active_power_flow_primary::Float64
+        reactive_power_flow_primary::Float64
+        active_power_flow_secondary::Float64
+        reactive_power_flow_secondary::Float64
+        active_power_flow_tertiary::Float64
+        reactive_power_flow_tertiary::Float64
+        r_primary::Float64
+        x_primary::Float64
+        r_secondary::Float64
+        x_secondary::Float64
+        r_tertiary::Float64
+        x_tertiary::Float64
+        rating::Union{Nothing, Float64}
+        r_12::Float64
+        x_12::Float64
+        r_23::Float64
+        x_23::Float64
+        r_13::Float64
+        x_13::Float64
+        g::Float64
+        b::Float64
+        primary_turns_ratio::Float64
+        secondary_turns_ratio::Float64
+        tertiary_turns_ratio::Float64
+        available_primary::Bool
+        available_secondary::Bool
+        available_tertiary::Bool
+        rating_primary::Float64
+        rating_secondary::Float64
+        rating_tertiary::Float64
+        services::Vector{Service}
+        ext::Dict{String, Any}
+        internal::InfrastructureSystemsInternal
+    end
+
+A 3-winding transformer.
+
+The model uses an equivalent star model with a star (hidden) bus. The user must transform the data to use `CW = CZ = CM = 1` and `COD1 = COD2 = COD3 = 0` (no voltage control) if taken from a PSS/E 3W transformer model. Three equivalent impedances (connecting each side to the star bus) are required to define the model. Shunt conductance (iron losses) and magnetizing susceptance can be considered from the star bus to ground. The model is described in Chapter 3.6 in J.D. Glover, M.S. Sarma and T. Overbye: Power Systems Analysis and Design.
```

Comment:
What is the expected behavior if the parser encounters a PSS/E transformer record where `CW = CZ = CM = 1` or `COD1 = COD2 = COD3 = 0` is violated? Will it automatically perform this transformation?

Reply:
Currently it gets ignored. We could fail, give a warning that it will be ignored, or add all the flags to the struct, and we can save them if we want, but will likely not be used anywhere. @jd-lara What do you think we should do here? We should probably add a lot of default fields with all the flags? So at least we can store them always and use them in our exporter.

Reply:
Currently being included in the EXT